Gemfile

```diff
@@ -18,7 +18,7 @@ group :lint, :development do
 end
 
 group :test, :development do
-  gem 'coveralls', require: false
+  gem 'coveralls_reborn', '~> 0.21.0', require: false
```

Comment:
I think we can allow _any_ version of coveralls_reborn.

Reply:
The fix was only introduced for v "0.21.0"  but I removed the explicit version as you recommended.